docs/source/quantization/fp8_e4m3_kvcache.rst

```diff
@@ -0,0 +1,38 @@
+.. _fp8_e4m3_kvcache:
+
+FP8 E4M3 KV Cache
+==================
+
+Quantizing the KV cache to FP8 reduces its memory footprint. This increases the number of tokens that can be stored in the 
+cache, improving throughput. OCP specifies two common floating point data formats: E5M2 (5 exponent bits and 2 mantissa 
+bits) and E4M3FN (4 exponent bits and 3 mantissa bits), often shortened to simply E4M3. One benefit of the E4M3 format
+over E5M2 is that floating point numbers are represented in higher precision. However, the small dynamic range of FP8 E4M3 
+(±240.0 can be represented) typically necessitates the use of a higher-precision (typically FP32) scaling factor alongside 
+each quantized tensor. For now, only per-tensor (scalar) scaling factors are supported. Development is ongoing to support 
+scaling factors of a finer granularity (e.g. per-channel).
+
+These scaling factors can be specified by passing an optional quantization param JSON to the LLM engine at load time. If 
+this JSON is not specified, scaling factors default to 1.0. These scaling factors are typically obtained when running an
+unquantized model through a quantizer tool (e.g. AMD quantizer or NVIDIA AMMO [pip install --no-cache-dir --extra-index-url https://pypi.nvidia.com nvidia-ammo]).
+
+Studies have shown that FP8 E4M3 quantization typically only minimally degrades inference accuracy. The most recent 
+silicon offerings e.g. AMD MI300, NVIDIA Hopper or later support native hardware conversion to/from fp32, fp16, bf16, etc. 
```

Comment:
MI300 or MI300x?

Reply:
to/from -> to and from

Reply:
In codebase, we use MI300 not to differentiate sub-models of MI300, though MI300X is a common SKU.

Reply:
solved